app/api/routes/product.py

```diff
@@ -1,148 +1,88 @@
-from fastapi import APIRouter, HTTPException, Depends,Query
-from uuid import UUID, uuid4
-from typing import Dict, Optional
-from pydantic import BaseModel, Field
-from dependencies import get_current_time,get_current_admin,products_db
-from models import Product,ProductCreate,ProductUpdate,ProductSearchParams
-
-
+from fastapi import APIRouter, Depends, status, HTTPException
+from sqlalchemy.orm import Session
```

Comment:
@Omarmanassrah @Mvamd12, did you test your changes? It looks like there are several missing requirements that are not included in the requirements.txt file. Have you tried running the service in Docker?

Reply:
For this I made some changes but it may be not enough my partner tell me that there some work on the project and that is not the final submit he want to make some changes today and he will push it and you will see if there is anything else to add.

Reply:
@Mohammed-samaheen  I'm waiting for this PR to be merged so I can align the requirements accordingly and take the necessary actions. Some behind the scenes changes have been completed, but I need this PR merged to finalize everything and ensure we're fully ready.